lib/nrf_modem_lib/lte_net_if/lte_net_if.c

```diff
@@ -413,6 +413,11 @@ static int lte_net_if_connect(struct conn_mgr_conn_binding *const if_conn)
 		return ret;
 	}
 
+	ret = lte_lc_modem_events_enable();
+	if (ret) {
+		LOG_WRN("lte_lc_modem_events_enable, error: %d", ret);
```

Comment:
Add a 

`return ret;`

Reply:
Do we want to return en error just because we're not getting some events from the modem? These events are only used for printing warnings. So I considered a failure to enable them isn't critical enough. Can you please explain why you think this is a critical error?

Reply:
I think it's fine to continue rather than return; it's not a critical error (in fact we print a warning).

Reply:
I see. I just expect any LTE_LC_* API to succeed at the connection establishment stage. If it fails and we ignore it, then something else is bound to fail at a later point. But since two of you consider this as non-critical failure, I am willing to leave it as is.